config/routes.php

```diff
@@ -24,7 +24,11 @@
 use Cake\Routing\Route\DashedRoute;
 use Cake\Routing\RouteBuilder;
 
-return static function (RouteBuilder $routes) {
+/**
```

Comment:
I think this should only be a simple multi-line comment:

```suggestion
/*
```